### DISCUSSION TITLE
Naming and Transferring Collaboration Document Tasks to the Project Board

### DISCUSSION THREAD

**Discussion by AlhassenSabeeh** [MIT-Emerging-Talent/ET6-foundations-group-28#9] [Ideas]
Dear Esteemed Teem Members, 
I hope this message finds you well. I propose that we document all the tasks we agreed upon during our second meeting, ensuring that each task is clearly labeled by its respective name as follows:

- Collaboration Readme
- Collaboration Communication
- Collaboration Constraints
- Collaboration Learning Goals

Subsequently, we should transfer these tasks to the project board and assign them to the "TODO" state. Each team member can then move their respective tasks to the "DOING" state as they work on them.
I would be happy to hear any additions or modifications to this approach.

**Comment by Adamx090** [comment]
Great idea. So should we create these tasks directly in project board, or make them as issues then move them to the board?

**Reply by AlhassenSabeeh** [reply]
To the best of my knowledge, it seems to me that every task we put in the Project Board should be linked to an issue. Thus, if the task falls within the issue categories in the Repo, we select the issue within its category first, where the issue can be discussed with the members of the team if needed. Then we include it in the Project Board. 
On the other  hand, if it does not make sense that this task can be categorized within the available issue categories, it can be created directly in the Project Board and then added  it to the group issues after its creation, as there are three dots next to the task name to be clicked and then converted to an issue to the group.
This is my suggestion for linking issues to tasks.

**Reply by Adamx090** [reply]
Correct, apparently when the task is added from the project board directly, it can be a draft that can be converted later into an issue or an issue from the start. I would also suggest that we include labels as much as we can, they will help organize our workflow better, and it will be easier for tracking. For instance I just label this discussion with enhancement as it's an idea to improve our workflow, this can be done for issues also.
